package.json

```diff
@@ -90,11 +90,11 @@
     }
   },
   "build": {
-    "artifactName": "${name}-${version}-${os}.${ext}",
     "files": [
-      "**/*"
+      "**/*",
+      "!media${/*}"
```

Comment:
What is this change for?

Reply:
So the `media` folder won't be included in the build.